SocialFilter/sections/popups.txt

```diff
@@ -388,6 +388,8 @@ hardreset.info#$##modal-facebook { display: none !important; }
 hardreset.info#$#.modal-backdrop { display: none !important; }
 hardreset.info#$#body { overflow: visible!important; }
 !
+!+ PLATFORM(ios, ext_android_cb, android)
```

Comment:
why for these platforms?

Reply:
`.social-icon ` bar only appears for mobile layout.